nucliadb_protos/resources.proto

```diff
@@ -291,6 +291,8 @@ message UserMetadata {
 message TokenSplit {
     string token = 1;
     string klass = 2;
+    uint32 start = 3;
+    uint32 end = 4;
```

Comment:
What will happen with already existing tokens that do not have `start` and `end` values set?

Reply:
seems like proto2 will assign a default value of 0